src/game/magic/spells/SpellsLvl06.cpp

```diff
@@ -460,9 +460,7 @@ void SlowDownSpell::End() {
 	m_targets.clear();
 }
 
-void SlowDownSpell::Update() {
-	
-}
```

Comment:
There seems to be more going on than what the commit message says. Please split formatting changes into a separate commit.

Reply:
I have restored the whitespace here and a few other places as it's not necessary for this PR. I'll try to split my commits in the future according to this.